test/source/from-test.js

```diff
@@ -0,0 +1,24 @@
+require('buster').spec.expose();
+var expect = require('buster').expect;
+
+var from = require('../../lib/source/from').from;
+var observe = require('../../lib/combinator/observe').observe;
+
+var sentinel = { value: 'sentinel' };
+var other = { value: 'other' };
+
+describe('from', function() {
+
+	it('should support array-like items', function() {
+		function observeArguments(){
+			var result = [];
+			return observe(function(x) {
+				result.push(x);
+			}, from(arguments)).then(function() {
+				expect(result).toEqual(result);
```

Comment:
This may not be testing what you think it is :)  I think it will always pass even if `from` is broken.  It's mutating `result`, and ends up comparing `result` with itself.  It should probably compare `result` with a known expected (and immutable :) ) array.